### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by christianadebambo**
add admin index page, create post page and css

### CODE REVIEW

**Comment by christianadebambo** [comment]
hi guys, please give reviews and suggestions.

File: home/templates/admin_create.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (Emmastro) [inline home/templates/admin_create.html:3]: Can you use the base.html, and inherit its content and style here? You can check how it's being used on the other templates.

File: home/templates/admin_index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```
Comment (Emmastro) [inline home/templates/admin_index.html:1]: same as the admin_create, it should include the base.html file.

File: home/static/css/admin_style.css [hunk 3]
```diff
@@ -0,0 +1,257 @@
+* {
```
Comment (Emmastro) [inline home/static/css/admin_style.css:1]: Can we consolidate the styles, some of the style you have here, or at least similar style, have been written by other members. Just to keep it all consistent, you can refactor and reuse the styles from the existing .css, and only keep here the style specific to the admin page if any

**Review by Emmastro — CHANGES_REQUESTED** [review]
@christianadebambo, can you add view functions on the views.py file, and a URL on the urls.py file, and test it running the django server? I have also left some comments on the code, but it looks good, just a few changes to make

**Pull request closed without merging**